lib/buildEmbed.js

```diff
@@ -10,6 +10,20 @@ module.exports = function(id, options, index) {
   return output;
 }
 
+function urlParams(options){
+  let params = []
+
+  // autoplay is _technically_ possible, but be cool, don't do this
+  if(options.allowAutoplay) params.push('autoplay=1');
+  if(options.recommendChannel) params.push('rel=0');
+  if(options.modestBranding) params.push('modestbranding=1');
+
+  if(params.length == 0){
+    return !1;
+  }
+  return params.join("&")
```

Comment:
- Any reason for returning `!1` instead of just `false`?
- Do we need to return anything? We could leave it and the assignment remains `undefined`.
- Just to be on the safe side I'd like to join the params with `&amp;`

```suggestion
if(params.length > 0) {
  return params.join("&amp;")
}
```